packages/gatsby-source-vtex/package.json

```diff
@@ -1,25 +1,28 @@
 {
-  "name": "@vtex/gatsby-source-vtex",
-  "version": "0.371.11",
-  "description": "Gatsby source plugin for building websites using VTEX as a data source.",
-  "main": "index.js",
-  "scripts": {
-    "build": "tsc",
-    "prepare": "cross-env NODE_ENV=production yarn run build",
-    "develop": "tsc --watch"
+  "name": "gatsby-source-vtex",
```

Comment:
why are you going to remove the prefix? `@vtex`

Reply:
oops, thanks!